src/coreclr/jit/optimizer.cpp

```diff
@@ -5542,14 +5542,30 @@ Compiler::fgWalkResult Compiler::optIsVarAssgCB(GenTree** pTree, fgWalkData* dat
 {
     GenTree* tree = *pTree;
 
-    if (tree->OperIs(GT_ASG))
+    if (tree->OperIsSsaDef())
     {
-        GenTree*   dest     = tree->AsOp()->gtOp1;
-        genTreeOps destOper = dest->OperGet();
-
         isVarAssgDsc* desc = (isVarAssgDsc*)data->pCallbackData;
         assert(desc && desc->ivaSelf == desc);
 
+        GenTree* dest = nullptr;
+        if (tree->OperIs(GT_CALL))
+        {
+            dest = tree->AsCall()->GetLclRetBufArgNode();
+            if (dest == nullptr)
+            {
+                desc->ivaMaskCall = optCallInterf(tree->AsCall());
+                return WALK_CONTINUE;
```

Comment:
This is a change in behavior: `optCallInterf` is now not called for calls with `GetLclRetBufArgNode() != nullptr`.

(This is the cause of the "big" regressions in the diffs - cloning kicks in where it didn't previously)

I have a suspicion that much of this "call interference" stuff is actually dead code, but it'd be best to leave it alone for this change.

Reply:
This is by mistake, not sure why I had it that way. The `ivaMaskCall` should have been outside.